configparser.go

```diff
@@ -17,7 +17,7 @@ const (
 )
 
 var (
-	sectionHeader = regexp.MustCompile(`\[([^]]+)\]`)
+	sectionHeader = regexp.MustCompile(`^\[([^]]+)\]$`) 
```

Comment:
Looks like a wee bit extra whitespace here, `gofmt -w configparser.go` should tidy this up.